app/uk/gov/hmrc/gatekeeper/views/emails/EmailPreferencesSelectedApiTopicView.scala.html

```diff
@@ -46,7 +46,7 @@ <h1 id="pageTitle" class="govuk-heading-l">Email users interested in a specific
                         @if(users.nonEmpty) {
                             <div class="govuk-button-group govuk-!-margin-bottom-4">
                                 <div style="visibility: visible">
-                                    <form method="post" action="@applicationConfig.gatekeeperComposeEmailUrl">
+                                    <form name="gatekeeper-compose-email" method="post" action="@applicationConfig.gatekeeperComposeEmailUrl">
```

Comment:
Can the `name` before more like a noun?  Maybe `gatekeeper-email-editor`?

Reply:
Addressed